go/test/endtoend/vtgate/foreignkey/fk_fuzz_test.go

```diff
@@ -555,58 +559,61 @@ func TestFkFuzzTest(t *testing.T) {
 		insertShare    int
 		deleteShare    int
 		updateShare    int
-	}{
-		{
-			name:           "Single Thread - Only Inserts",
-			concurrency:    1,
-			timeForTesting: 5 * time.Second,
-			maxValForCol:   5,
-			maxValForId:    10,
-			insertShare:    100,
-			deleteShare:    0,
-			updateShare:    0,
-		},
-		{
-			name:           "Single Thread - Balanced Inserts and Deletes",
-			concurrency:    1,
-			timeForTesting: 5 * time.Second,
-			maxValForCol:   5,
-			maxValForId:    10,
-			insertShare:    50,
-			deleteShare:    50,
-			updateShare:    0,
-		},
-		{
-			name:           "Single Thread - Balanced Inserts and Updates",
-			concurrency:    1,
-			timeForTesting: 5 * time.Second,
-			maxValForCol:   5,
-			maxValForId:    10,
-			insertShare:    50,
-			deleteShare:    0,
-			updateShare:    50,
-		},
-		{
-			name:           "Single Thread - Balanced Inserts, Updates and Deletes",
-			concurrency:    1,
-			timeForTesting: 5 * time.Second,
-			maxValForCol:   5,
-			maxValForId:    10,
-			insertShare:    50,
-			deleteShare:    50,
-			updateShare:    50,
-		},
-		{
-			name:           "Multi Thread - Balanced Inserts, Updates and Deletes",
-			concurrency:    30,
-			timeForTesting: 5 * time.Second,
-			maxValForCol:   5,
-			maxValForId:    30,
-			insertShare:    50,
-			deleteShare:    50,
-			updateShare:    50,
-		},
-	}
+	}{{
+		name:           "Single Thread - Only Inserts",
+		concurrency:    1,
+		timeForTesting: 5 * time.Second,
+		maxValForCol:   5,
+		maxValForId:    10,
+		insertShare:    100,
+		deleteShare:    0,
+		updateShare:    0,
+	}, {
+		name:           "Single Thread - Balanced Inserts and Deletes",
+		concurrency:    1,
+		timeForTesting: 5 * time.Second,
+		maxValForCol:   5,
+		maxValForId:    10,
+		insertShare:    50,
+		deleteShare:    50,
+		updateShare:    0,
+	}, {
+		name:           "Single Thread - Balanced Inserts and Updates",
+		concurrency:    1,
+		timeForTesting: 5 * time.Second,
+		maxValForCol:   5,
+		maxValForId:    10,
+		insertShare:    50,
+		deleteShare:    0,
+		updateShare:    50,
+	}, {
+		name:           "Single Thread - Balanced Inserts, Updates and Deletes",
+		concurrency:    1,
+		timeForTesting: 5 * time.Second,
+		maxValForCol:   5,
+		maxValForId:    10,
+		insertShare:    50,
+		deleteShare:    50,
+		updateShare:    50,
+	}, {
+		name:           "Multi Thread - Only Inserts",
+		concurrency:    30,
+		timeForTesting: 5 * time.Second,
+		maxValForCol:   5,
+		maxValForId:    30,
+		insertShare:    100,
+		deleteShare:    0,
+		updateShare:    0,
+	}, {
+		name:           "Multi Thread - Balanced Inserts, Updates and Deletes",
+		concurrency:    30,
+		timeForTesting: 5 * time.Second,
+		maxValForCol:   5,
+		maxValForId:    30,
+		insertShare:    50,
+		deleteShare:    50,
+		updateShare:    50,
+	}}
```

Comment:
Seems like this test is failing on CI:
```
    --- FAIL: TestFkFuzzTest/Unsharded_-_Multi_Thread_-_Balanced_Inserts,_Updates_and_Deletes_QueryFormat_-_PreparedStatementPacket (4.78s)
```

Reply:
The failure looks unrelated to this PR. Investigating it.

Reply:
PR that fixes the issue https://github.com/vitessio/vitess/pull/14524